src/cr/cube/cube.py

```diff
@@ -645,6 +705,15 @@ class _UnweightedCountMeasure(_BaseMeasure):
     @lazyproperty
     def _flat_values(self):
         """tuple of int counts before weighting."""
+        # ---If valid_count are expressed in the cube dict, returns its data---
```

Comment:
This is great. Maybe put it _under_ the if statement (I know it's generally supposed to be above it, but I think here it will serve us better if it's under). And also, add some more context, like, when this might happen and why (for our future selves).